apps/native/app/src/ui/lib/problem/problem-template.tsx

```diff
@@ -119,12 +121,10 @@ export const ProblemTemplate = ({
   return (
     <Host borderColor={borderColor} noContainer={withContainer}>
       {tag && (
-        <Tag
-          variant="eyebrow"
-          backgroundColor={tagBackgroundColor}
-          color={tagColor}
-        >
-          {tag}
+        <Tag backgroundColor={tagBackgroundColor}>
```

Comment:
I was adding this and testing how it would display and then the `border-radius` was not being applied to the Tag, after googling I found that` border-radius` cannot be added to `Text` elements but I am sure this was looking okay some time ago so I don't know what happened 😅 At least I fixed it for now.